src/main/kotlin/com.bugsnag.android.gradle/Call.kt

```diff
@@ -0,0 +1,48 @@
+package com.bugsnag.android.gradle
+
+import org.gradle.api.Project
+
+internal abstract class Call protected constructor(// 60 seconds
```

Comment:
Is this `// 60 seconds` comment still relevant? I'm not sure what it refers to :s

Reply:
Doesn't seem to be relevant - I've removed it from the changeset.